Cards/CardContainer.gd

```diff
@@ -61,11 +61,16 @@ func set_queued_card(card: CardWorld) -> void:
 
 
 func remove_queued_card() -> void:
-	_focused_card = null
-	discard_card(queued_card)
+	_cards_queued_for_discard.append(queued_card)
+	_remove_queued_card_from_hand()
 	set_queued_card(null)
```

Comment:
I think we need to add a `_focused_card = null` here, because I noticed a bug in `_update_card_positions` where the hand does a bit of strange movement due to the `_focused_card` still being set whenever we move the active card to the discard pile